### PR TITLE
Andrek/standardize

### PR DESCRIPTION
**Pull request by wpschaub**
Andre, here is your DRAFT post. I linked your references to our CoE events to our blog posts and picked 04/06 as the publication day.

Can you send me your mugshot and bio, so that I can set that up as well?

### CODE REVIEW

**Review by ShayVannery — DISMISSED** [review]

**Comment by wpschaub** [comment]
@AndreKaminski your revisions have been applied.

**Review by AndreKaminski — CHANGES_REQUESTED** [review]
There is a spelling mistake in my name in the slug

**Review by ShayVannery — DISMISSED** [review]

**Review by erwsch1 — DISMISSED** [review]

File: content/standardize-or-not-to-standardize.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+Title: Common Engineering - To Standardize or Not to Standardize, view from a directors point
+Date: 2022-04-06
+Category: Posts
+Tags: journal, innovation
+Slug: standardize-or-not-to-standardize
+Author: Andre Kaminsiki
```
Comment (AndreKaminski) [inline content/standardize-or-not-to-standardize.md]: Spelling mistake - should be 'Kaminski'

**Review by AndreKaminski — CHANGES_REQUESTED** [review]
spelling mistake in the slug

File: content/standardize-or-not-to-standardize.md (hunk 1 above)
Comment (wpschaub) [reply content/standardize-or-not-to-standardize.md]: @AndreKaminski ... My bad and appologies! Fixed.

I was actually staring at it yesterday and it felt wrong, but my brain did not compute.

**Review by erwsch1 — APPROVED** [review]

**Review by AndreKaminski — APPROVED** [review]

**Pull request merged**